tests/Model/IndexInfoFunctionalTest.php

```diff
@@ -14,17 +14,8 @@ public function setUp(): void
     {
         parent::setUp();
 
+        $this->dropCollection($this->getDatabaseName(), $this->getCollectionName());
```

Comment:
This would be consistent with what you did above in ChangeStreamIteratorTest.

```suggestion
        $this->collection = $this->dropCollection($this->getDatabaseName(), $this->getCollectionName());
```

Note: I couldn't remove the following line in the suggestion since it wasn't modified in the PR.